apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ConsumerController.java

```diff
@@ -77,12 +100,21 @@ public ConsumerToken getConsumerTokenByAppId(@RequestParam String appId) {
     return consumerService.getConsumerTokenByAppId(appId);
   }
 
+  @PreAuthorize(value = "@permissionValidator.isSuperAdmin()")
+  @GetMapping(value = "/consumer/info/by-appId")
+  public ConsumerInfo getConsumerInfoByAppId(@RequestParam String appId) {
+    return consumerService.getConsumerInfoByAppId(appId);
+  }
+
   @PreAuthorize(value = "@permissionValidator.isSuperAdmin()")
   @PostMapping(value = "/consumers/{token}/assign-role")
-  public List<ConsumerRole> assignNamespaceRoleToConsumer(@PathVariable String token,
-                                                          @RequestParam String type,
-                                                          @RequestParam(required = false) String envs,
-                                                          @RequestBody NamespaceDTO namespace) {
+  public List<ConsumerRole> assignNamespaceRoleToConsumer(
+      @PathVariable String token,
+      @RequestParam String type,
+      @RequestParam(required = false) String envs,
+      @RequestBody NamespaceDTO namespace
+  ) {
```

Comment:
It seems the code style has changed slightly?
```suggestion
      @RequestBody NamespaceDTO namespace) {
```